rs/macros/core/src/program/mod.rs

```diff
@@ -206,9 +206,19 @@ impl ProgramBuilder {
 
         let solidity_main = self.sol_main(solidity_dispatchers.as_slice());
 
+        let accept_transfer = self.program_args.accept_transfer().then(|| {
+            quote! {
+                if #sails_path::gstd::msg::value() > 0 && #sails_path::gstd::msg::size() == 0 {
+                    return;
+                }
+            }
+        });
+
         let main_fn = quote!(
             #[gstd::async_main #async_main_args]
```

Comment:
Shouldn't it be like this?
```suggestion
            #[#sails_path::gstd::async_main #async_main_args]
```

Reply:
I'd double check all entries in future

Reply:
seems test-able

Reply:
It's ok, use sails_rs::gstd in the module.
```
pub mod wasm {
    use super::*;
    use sails_rs::{gstd, hex, prelude::*};
```